cmd/bosun/web/static/js/directives.ts

```diff
@@ -218,8 +218,11 @@ bosunApp.directive('tsTimeLine', () => {
 	return {
 		link: (scope: any, elem: any, attrs: any) => {
 			scope.shown = {};
-			scope.collapse = (i: any) => {
+			scope.collapse = (i: any, entry:any, v: any) => {
 				scope.shown[i] = !scope.shown[i];
+				if (scope.loadTimelinePanel && entry && scope.shown[i]){
```

Comment:
when does `scope.loadTimelinePanel` not exist? why do we need to check for it here?